src/goDebugConfiguration.ts

```diff
@@ -24,7 +24,7 @@ export class GoDebugConfigurationProvider implements vscode.DebugConfigurationPr
 				type: 'go',
 				request: 'launch',
 				mode: 'auto',
-				program: '${fileDirname}',
+				program: '${file}',

```

Comment:
I understand this change is based on my comment https://github.com/microsoft/vscode-go/issues/1229#issuecomment-473731132

But I now realize that this change can break the workflow for existing users who might 
start debugging from a file that does not have the main function. Having `${fileDirname}` ensured that such scenarios are supported.

I recommend we not make this change. Folks wanting to have multiple files with main function in the same folder, can create a debug configuration with ${file} and have the debugging feature work for them